src/stores/tournament.store.ts

```diff
@@ -247,6 +247,20 @@ export const useTournamentStore = defineStore('tournament', () => {
     window.location.href = redirect_url;
   }
 
+  async function patch_registration(registration_type: string, registration_id: number, data: Record<string, string>) {
+    await get_csrf();
+
+    await axios.patch(`/tournament/${registration_type}/${registration_id}/`, data, {
+      withCredentials: true,
+      headers: {
+        'X-CSRFToken': csrf.value,
+      },
+    });
+
+    // reload the page
+    window.location.reload();
```

Comment:
Might be better to update the tournament store rather than reloading the whole page.